### ISSUE TITLE
generated code involving `default` not parsable by acorn

### ISSUE THREAD

**Issue by kellyselden** [buxlabs/amd-to-es6#70]
input:

```js
define('ember-inflector/lib/helpers/pluralize', ['exports', 'ember-inflector', 'ember-inflector/lib/utils/make-helper'], function (exports, _emberInflector, _makeHelper) {
  'use strict';

  Object.defineProperty(exports, "__esModule", {
    value: true
  });
  exports.default = (0, _makeHelper.default)(function (params, hash) {
    var count = void 0,
        word = void 0,
        withoutCount = false;

    if (params.length === 1) {
      word = params[0];
      return (0, _emberInflector.pluralize)(word);
    } else {
      count = params[0];
      word = params[1];

      if (hash["without-count"]) {
        withoutCount = hash["without-count"];
      }

      if (parseFloat(count) !== 1) {
        word = (0, _emberInflector.pluralize)(word);
      }

      return withoutCount ? word : count + " " + word;
    }
  });
});
```

output:

```js
import _emberInflector from 'ember-inflector';
import _makeHelper from 'ember-inflector/lib/utils/make-helper';
Object.defineProperty(exports, '__esModule', { value: true });
export var default = (0, _makeHelper.default)(function (params, hash) {
    var count = void 0, word = void 0, withoutCount = false;
    if (params.length === 1) {
        word = params[0];
        return (0, _emberInflector.pluralize)(word);
    } else {
        count = params[0];
        word = params[1];
        if (hash['without-count']) {
            withoutCount = hash['without-count'];
        }
        if (parseFloat(count) !== 1) {
            word = (0, _emberInflector.pluralize)(word);
        }
        return withoutCount ? word : count + ' ' + word;
    }
});
```

Acorn and most other parsers fail on the `export var default` code.

**Comment by emilos** [comment]
hey @kellyselden thanks for the example :) definitely a bug, I'll try to provide a fix soon

**Comment by kellyselden** [comment]
@emilos Much appreciated!

**Comment by emilos** [comment]
@kellyselden the fixes are in the `0.11.0` version, please let me know if it helps https://github.com/buxlabs/amd-to-es6/commit/2bfd7e991f1b6b9ca149d4c0dca6bcb92b1bcaee

**Comment by kellyselden** [comment]
This is now fixed. Thanks!
